.env.production

```diff
@@ -1,31 +0,0 @@
-#defaults for build based on dev env
```

Comment:
i think this was required for vercel defaults
test deployment on vercel after deployment

Reply:
ok